src/static/css/variables.css

```diff
@@ -0,0 +1,79 @@
+:root {
+    /* Colors */
+    --color-blue-100-hsl: 234, 100%, 96%;
+    --color-blue-100: hsl(var(--color-blue-100-hsl));
+    --color-blue-500-hsl: 234, 62%, 51%;
+    --color-blue-500: hsl(var(--color-blue-500-hsl));
+    --color-blue-700-hsl: 234, 61%, 33%;
+    --color-blue-700: hsl(var(--color-blue-700-hsl));
+    --color-green-100-hsl: 170, 37%, 97%;
+    --color-green-100: hsl(var(--color-green-100-hsl));
+    --color-green-300-hsl: 170, 54%, 89%;
+    --color-green-300: hsl(var(--color-green-300-hsl));
+    --color-green-500-hsl: 175, 71%, 69%;
+    --color-green-500: hsl(var(--color-green-500-hsl));
+    --color-green-700-hsl: 175, 80%, 16%;
+    --color-green-700: hsl(var(--color-green-700-hsl));
+    --color-red-100-hsl: 0, 100%, 97%;
+    --color-red-100: hsl(var(--color-red-100-hsl));
+    --color-red-500-hsl: 0, 89%, 69%;
+    --color-red-500: hsl(var(--color-red-500-hsl));
+    --color-orange-100-hsl: 40, 100%, 97%;
+    --color-orange-100: hsl(var(--color-orange-100-hsl));
+    --color-orange-500-hsl: 40, 100%, 49%;
+    --color-orange-500: hsl(var(--color-orange-500-hsl));
+
+    /* Fonts */
+    --font-sans:
+        martel,
+        -apple-system,
+        blinkmacsystemfont,
+        avenir next,
+        avenir,
+        segoe ui,
+        helvetica neue,
+        helvetica,
+        cantarell,
+        ubuntu,
+        roboto,
+        noto,
+        arial,
+        sans-serif;
+    --font-serif:
+        mulish,
+        iowan old style,
+        apple garamond,
+        baskerville,
+        times new roman,
+        droid serif,
+        times,
+        source serif pro,
+        apple color emoji,
+        segoe ui emoji,
+        segoe ui symbol;
+
+    /* Spacings */
```

Comment:
Can you add the link comment from the Utopia generator for reference before all the fluid variables? Looks like this I think:

```css
/* @link https://utopia.fyi/space/calculator?c=320,18,1.2,1240,20,1.25,5,2,&s=0.75|0.5|0.25,1.5|2|3|4|6,s-l&g=s,l,xl,12 */
```

Reply:
Sorry, I should have clarified— I was suggesting the link to the specific configuration you used (which has query parameters on the end of it).

Also, the link should appear before the `/* Spacing */` comment as it is a header for all of the remaining blocks.